jira/client.py

```diff
@@ -210,59 +210,63 @@ class JiraCookieAuth(AuthBase):
     """Jira Cookie Authentication
 
     Allows using cookie authentication as described by
-    https://developer.atlassian.com/jiradev/jira-apis/jira-rest-apis/jira-rest-api-tutorials/jira-rest-api-example-cookie-based-authentication
-
+    https://developer.atlassian.com/server/jira/platform/cookie-based-authentication/
     """
 
     def __init__(
-        self, session: ResilientSession, _get_session: Callable, auth: Tuple[str, str]
+        self, session: ResilientSession, session_api_url: str, auth: Tuple[str, str]
```

Comment:
im not aware why this got broken but just a reminder that this breaks the API as probably mentioned here https://github.com/pycontribs/jira/pull/1201#issuecomment-955017866 by @mruessler .

maybe this should become a private class if we dont want people to interact with it?

Reply:
I'll wait for @mruessler to respond to my earlier comment. I agree this is private. I can't imagine why a user would directly interact with this class, all these 'AuthBase' subclasses need to be in a separate module file with a module docstring describing them as an implementation detail for the main client.